src/validations/class.js

```diff
@@ -4,10 +4,12 @@ const validateCreation = (req, res, next) => {
   const classValidation = Joi.object({
     activity: Joi.string().hex().length(24).required(),
     trainer: Joi.string().hex().length(24).required(),
-    day: Joi.string().regex(/^(Monday|Tuesday|Wednesday|Thursday|Friday|Saturday|Sunday)$/).required(),
-    time: Joi.string().regex(/^([0-9]|[01]\d|2[0-3]):([0-5]\d)$/).required(),
+    day: Joi.string().regex(/^(Monday|Tuesday|Wednesday|Thursday|Friday|Saturday)$/).required(),
+    time: Joi.string().regex(/^([0-9]|[01]\d|2[0-3]):([00]\d)$/).required(),
     capacity: Joi.number().min(1).max(50).required()
       .integer(),
+    subscribed: Joi.number().min(0).max(50).integer(),
```

Comment:
if necessary, add that data is required

```suggestion
    subscribed: Joi.number().min(0).max(50).integer().required(),
```